ert_shared/ensemble_evaluator/evaluator.py

```diff
@@ -247,46 +242,69 @@ def stop(self):
         self._loop.call_soon_threadsafe(self._stop)
         self._ws_thread.join()
 
+    def _signal_cancel(self):
```

Comment:
`_handle_cancel` as it doesn't signal anything, but rather responds to a signal?

Reply:
I beg to differ...  😄   This code just forwards the desire to cancel evaluation, returning immediately (hence `_signal` ). I'm open to rename it but it does not primarily respond to a signal.